packages/demoing-storybook/src/start/transformers/createServeStorybookTransformer.js

```diff
@@ -1,15 +1,53 @@
-module.exports = function createServeStorybookTransformer(assets) {
-  return function serveStorybookTransformer({ url }) {
+const { injectStories } = require('../../shared/injectStories');
+const { createOrderedExports } = require('../../shared/createOrderedExports');
+
+/**
+ * @param {object} args
+ * @param {object} args.assets
+ * @param {string} args.assets.indexHTML
+ * @param {string} args.assets.iframeHTML
+ * @param {string} args.previewImport
+ * @param {string[]} args.storiesPatterns
+ * @param {string} args.rootDir
+ */
+function createServeStorybookTransformer({ assets, previewImport, storiesPatterns, rootDir }) {
+  const servedStoryUrls = new Set();
+  const { indexHTML, iframeHTML } = assets;
```

Comment:
nit: can we destructure this in params?

Reply:
I don't like it when a function declaration goes multiline, it's hard to see where things start and end.